bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GroupItem.java

```diff
@@ -286,7 +287,9 @@ public void stateChanged(Item item, State oldState, State newState) {
     @Override
     public void stateUpdated(Item item, State state) {
         State oldState = this.state;
-        setState(function.calculate(members));
+        if (null != function) {

```

Comment:
I know, it's a matter of personal taste, but so far we have them the other way round pretty consistently.